src/GlobalCtx.tsx

```diff
@@ -7,6 +7,7 @@ export type Ctx = {
   registry: TypeRegistry;
   wsAddress: string;
   latestFinalizedBlock: number | null;
+  latestFinalizedBlockTimestamp: Date | null;
```

Comment:
This is slightly better
```suggestion
  latestFinalizedBlock: {blockNumber: number, timestamp: Date } | null;
```